root-common/usr/share/container-scripts/mysql/init/40-datadir-action.sh

```diff
@@ -5,6 +5,7 @@ upstream_upgrade_info() {
     10.1) echo "https://mariadb.com/kb/en/library/upgrading-from-mariadb-100-to-mariadb-101/" ;;
     10.2) echo "https://mariadb.com/kb/en/library/upgrading-from-mariadb-101-to-mariadb-102/" ;;
     10.3) echo "https://mariadb.com/kb/en/library/upgrading-from-mariadb-102-to-mariadb-103/" ;;
+    10.5) echo "https://mariadb.com/kb/en/upgrading-from-mariadb-104-to-mariadb-105/" ;;
```

Comment:
We are not planning to support 10.4 in https://github.com/sclorg/mariadb-container. Therefore this is not valid, my thinking. Does it make sense to add there also upgrade from 10.3 to 10.4, so users are informed properly?

Reply:
It does make sense, thanks for the suggestion. Fixing.